04/app.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded", () => {
+    const clickActions = {
+        addClicked: el => setTimeout(() => el.classList.add("clicked"), parseInt(el.dataset.time, 10) || 0),
+        removeClicked: el => el.classList.remove("clicked")
+    };
+
+    const registerClick = (el, action) => el.addEventListener("click", e => action(e.currentTarget));
```

Comment:
Może spróbujesz zrobić jeszcze inaczej. Jeśli w tralcie "zakolorywania" prostokątów klikę w `body` to zaczymuje kolorowanie. Przykład. Klikam w środkowy prostokąt. Zakolorował się, następnie klikam w body i mam zatrzymać resztę kolorowania. Obecnie zakolorują (wypełnią) się pozostałe.